client/state/selectors/test/get-site-url.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteUrl } from '..';
+
+describe( 'getSiteUrl()', () => {
+	it( 'should return null if the site is unknown', () => {
+		const state = {
+			sites: {
+				items: {}
+			}
+		};
+
+		expect( getSiteUrl( state ) ).to.be.null;
+		expect( getSiteUrl( state, 123 ) ).to.be.null;
+	} );
+
+	it( 'should return the Url for a site', () => {
+		const URL = 'https://wordpress.com';
```

Comment:
Nitpicky nit: it's worth to mention that this `const` doesn't match our variable naming standards.

I'm fine with it that way, as it simplifies the the test state definition code. Your call 😉